bld/namelist_files/use_cases/hist_geoschem_nudged.xml

```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0"?>
+
+<namelist_defaults>
+
+<!-- Settings in this file were adapted from hist_trop_strat_vbs_cam6.xml -->
+
+<!-- Settings for using GEOS-Chem -->
+
+<gc_cheminputs>/glade/p/univ/umit0034/ExtData/CHEM_INPUTS/</gc_cheminputs>
+
+<ncdata dyn="fv"  hgrid="0.9x1.25">/glade/p/univ/umit0034/Shared/GEOS-Chem/f.e20.FC2010.f09_f09.144.GC_vbsext.001.cam.i.0007-01-01-00000.nc</ncdata>
+
+<ncdata dyn="fv"  hgrid="1.9x2.5">/glade/p/univ/umit0034/Shared/GEOS-Chem/f.e20.FC2010.f19_f19.144.GC_vbsext.001.cam.i.0007-01-01-00000.nc</ncdata>
+
+<dep_data_file>/glade/u/home/elundgren/drydep_henrys_law_nc/dep_data_file_geoschem.nc</dep_data_file>
+
+<!-- Settings mostly in common with hist_trop_strat_vbs_cam6.xml -->
+
+<start_ymd>00010101</start_ymd>
+
+<!-- Solar data -->
+<solar_irrad_data_file>atm/cam/solar/SolarForcingCMIP6_18491230-23000102_c20200615.nc</solar_irrad_data_file>
+
+<!-- WACCM GW Settings -->
+<use_gw_front>.true.</use_gw_front>
+<use_gw_convect_dp>.true.</use_gw_convect_dp>
+<tau_0_ubc>.false.</tau_0_ubc>
+<gw_qbo_hdepth_scaling>0.25D0</gw_qbo_hdepth_scaling>
+
+<flbc_type>SERIAL</flbc_type>
+<flbc_file>atm/waccm/lb/LBC_1750-2014_CMIP6_0p5degLat_c170126.nc</flbc_file>
+
+<prescribed_strataero_type>SERIAL</prescribed_strataero_type>
+
+<ndep_list>'noy', 'nhx'</ndep_list>
```

Comment:
The nudged file also contains these:
<ext_frc_type>INTERP_MISSING_MONTHS</ext_frc_type>
<srf_emis_type>INTERP_MISSING_MONTHS</srf_emis_type>
I know we don't need these for GEOS-Chem but could you remind me why?

Reply:
I think these are used for the CAM-chem's offline emissions module. We use HEMCO so these are just placeholders. Eventually when HEMCO is default all these XML options will be removed.

Reply:
Got it, thanks!